package.json

```diff
@@ -19,8 +19,8 @@
         "type": "git",
         "url": "https://github.com/Microsoft/TypeScript.git"
     },
-    "main": "./lib/typescript.js",
-    "typings": "./lib/typescript.d.ts",
+    "main": "./built/local/typescript.js",
+    "typings": "./built/local/typescript.d.ts",
```

Comment:
Note: this will have to be removed. FWIW these days `hereby lkg` does not modify the tree and so you're safe to run it and use the package.

Reply:
This was left over fro when we had those extra packages. Reverted.